reference/library/src/morpheus-master/sass/modules/timeout-alert/_base.scss

```diff
@@ -1,3 +1,10 @@
 #timeout_alert_body{
-
+	z-index: 1001;
+	background: $background-color;
+	position: absolute;
+	top: 10em;
+	left: 10em;
+	width: calc(100% - 20em);
```

Comment:
I can add a fallback in all calcs I have added in morpheus using a mixin instead a property.

From experience, do you think we'll have too much problems? by now, our users have had no problems with it but surely we should think in every user in Sakai.